src/routes/articles/like-article.e2e.test.js

```diff
@@ -0,0 +1,129 @@
+const {expect} = require('chai')
+const {fixtures} = require('../../../test/utils')
+
+
+describe('POST /articles/:id/likes', () => {
+
+	let user
+	let accessToken
+
+	before(async ()=>{
+
+		 user = await fixtures.insertUser() 
+		const body = { id: user.id, email: user.email }
+		accessToken = fixtures.generateAccessToken(
+			{user : body}
+		)
+	})
+
+	describe('Failure', () => {
+		
+		let data
+		before(async ()=>{
+			data = {
+				userId : user.id
+			}
+		})
+
+		it('should return 400 if postId is not a number', async () => {
+			const expectedError = {
+				"error": {
+			  "message": "id must be a number"
+				},
+		   "status": "failed"
+		  }
+			return	fixtures.api()
+				.post(`/api/v1/articles/sdsdsd/likes`)
+				.set('Authorization', `Bearer ${accessToken}`)
+				.send(data)
+				.expect(400, expectedError)
+
+		})
+
+		it('should return 401 if request is not authenticated', async () => {
+
+			const post = await fixtures.insertPost({
+				userId : user.id , 
+				type : 'article'
+			}) 
+			return fixtures.api()
+				.post(`/api/v1/articles/${post.id}/likes`)
+				.expect(401)
+
+		})
+
+		it('should throw an error if Article is already liked', async () => {
```

Comment:
unlike service tests, api/endpoint test are not really about throwing errors, mainly about status codes and response code

```suggestion
		it('should return 422 if article is already liked', async () => {
```

Reply:
awesome! thanks a lot